src/Liip/RMT/Action/BuildPharPackageAction.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/*
+ * This file is part of the project RMT
+ *
+ * Copyright (c) 2013, Liip AG, http://www.liip.ch
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\RMT\Action;
+
+use Phar;
+use FilesystemIterator;
+
+use Liip\RMT\Context;
+
+/**
+ * Builds a Phar package of the current project.
+ */
+class BuildPharPackageAction extends BaseAction
+{
+
+    protected $releaseVersion;
+
+    public function __construct($options)
+    {
+        $this->options = array_merge(array(
+            'package-name' => 'rmt-package',
+            'destination' => '/tmp/',
+            'excluded-paths' => '',
+            'metadata' => [],
```

Comment:
You're right.
But what about upping to PHP 5.4, especially considering http://php.net/supported-versions.php